.gitignore

```diff
@@ -77,3 +77,4 @@ sftp-config.json
 target/
 var/
 venv/
+backup/
```

Comment:
I added this as a storage for my db backups, as well as for my backup and restore sh scripts, but I can remove that if you like.